golang/cosmos/x/vbank/vbank.go

```diff
@@ -144,11 +144,9 @@ func (ch portHandler) Receive(cctx context.Context, str string) (ret string, err
 		}
 		coin := keeper.GetBalance(ctx, addr, msg.Denom)
 		packet := coin.Amount.String()
+		bytes, err := json.Marshal(&packet)
 		if err == nil {
-			bytes, err := json.Marshal(&packet)
-			if err == nil {
-				ret = string(bytes)
-			}
+			ret = string(bytes)
 		}
```

Comment:
the functions that return error return on error so there is no need to check here if error is == nil